ui/src/views/Weights.vue

```diff
@@ -2,55 +2,53 @@
 import axios from "axios";
 
 export default {
-  data() {},
+  data() {
+    return {
+      clientId: [],
+      clientWeights: [],
+      client_name: " ",
+    };
+  },
+
   mounted() {
     this.getWeights();
   },
 
   methods: {
-    getWeights() {
+    getWeights(clientId) {
```

Comment:
You're not using `clientId` here-- you can remove it

Reply:
done